gazebo_ros/scripts/libcommon.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# the function relocate all the remappings in the command at the end of the string
```

Comment:
"the function relocate" => "the function relocates"
"all the remappings" => "all the ROS remappings"